scripts/grafana-server/custom.ini

```diff
@@ -1,10 +1,15 @@
+app_mode = development
```

Comment:
Is this necessary for running the plugins? I'm not sure exactly what it does across the entire codebase but could it have unintended side-effects for testing against production instances? 🤔